apps/fluent-tester/src/RNTester/TestComponents/Checkbox/index.ts

```diff
@@ -1 +1 @@
-export * from './CheckboxTest';
+export * from './CheckboxTest';
```

Comment:
nit does our style say to end w/ new lines?

Reply:
Yes it seems so, I just looked around the repo. I will add the new lines back in.